src/Files.App/Views/Layouts/BaseLayoutPage.cs

```diff
@@ -1176,11 +1197,12 @@ private void RefreshItem(SelectorItem container, object item, bool inRecycleQueu
 			}
 		}
 
-		protected static void FileListItem_PointerPressed(object sender, PointerRoutedEventArgs e)
+		protected internal void FileListItem_PointerPressed(object sender, PointerRoutedEventArgs e)
 		{
 			if (sender is not SelectorItem selectorItem)
 				return;
 
+			SelectedItem = GetItemFromElement(sender);
```

Comment:
This can be limited to when an already selected item is clicked and not deselected.
```csharp
			if (selectorItem.IsSelected)
			{
				if (e.KeyModifiers == VirtualKeyModifiers.Control)
				{
					selectorItem.IsSelected = false;

					// Prevent issues arising caused by the default handlers attempting to select the item that has just been deselected by ctrl + click
					e.Handled = true;
				}
				else
				{
					SelectedItem = GetItemFromElement(sender);
				}
			}
			else if (e.GetCurrentPoint(selectorItem).Properties.IsLeftButtonPressed)
			{
				selectorItem.IsSelected = true;
			}
```